File: weathersdk/src/main/java/fr/ekito/myweatherlibrary/json/weather/Simpleforecast.java
package fr.ekito.myweatherlibrary.json.weather;

import com.google.gson.annotations.Expose;

import java.util.ArrayList;
import java.util.List;

public class Simpleforecast {

    @Expose
    private List<Forecastday_> forecastday = new ArrayList<Forecastday_>();

    /**
     * @return The forecastday
     */
    public List<Forecastday_> getForecastday() {
        return forecastday;
    }

    /**
     * @param forecastday The forecastday
     */
    public void setForecastday(List<Forecastday_> forecastday) {
        this.forecastday = forecastday;
    }

}

File: weathersdk/src/main/kotlin/fr/ekito/myweatherlibrary/json/weather/Simpleforecast.kt
package fr.ekito.myweatherlibrary.json.weather

import com.google.gson.annotations.Expose

import java.util.ArrayList

class Simpleforecast {

    /**
     * @return The forecastday
     */
    /**
     * @param forecastday The forecastday
     */
    @Expose
    var forecastday: List<Forecastday_> = ArrayList()

}
